bson/src/test/unit/org/bson/json/JsonReaderTest.java

```diff
@@ -979,7 +979,7 @@ public void testSkip() {
 
     @Test
     public void testDBPointer() {
-        String json = "DBPointer(\"b\",\"5209296cd6c4e38cf96fffdc\")";
+        String json = "DBPointer(\"b\",ObjectId(\"5209296cd6c4e38cf96fffdc\"))";
```

Comment:
This was a bug and not valid shell syntax

Reply:
Huh, wonder why we ever thought it was.